hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/services/CompareIndexService.kt

```diff
@@ -38,7 +38,7 @@ class CompareIndexService(
   private companion object {
     private val log: Logger = LoggerFactory.getLogger(this::class.java)
     private const val CUTOFF = 50
-    private val scroll = Scroll(TimeValue.timeValueMinutes(1L))
+    private val scroll = Scroll(TimeValue.timeValueHours(2L))
```

Comment:
This is how long it typically takes though Im not likely to be using this service going forward

Reply:
Dubious about this change.  The parameter here is how long to keep the scroll open between requests rather than the total time it takes to process the scroll.  From https://opensearch.org/docs/latest/api-reference/scroll/
> This search context needs to be long enough to process a single batch of results.

Reply:
Ah yes ok, ill change it